scanner/cmd/scannerctl/main.go

```diff
@@ -43,8 +50,12 @@ func main() {
 	c := v4.NewIndexerClient(conn)
 
 	resp, err := c.CreateIndexReport(ctx, &v4.CreateIndexReportRequest{
-		HashId:          "",
-		ResourceLocator: nil,
+		HashId: fmt.Sprintf("/v4/containerimage/%x", sha512.Sum512([]byte(imageURL))),
+		ResourceLocator: &v4.CreateIndexReportRequest_ContainerImage{ContainerImage: &v4.ContainerImageLocator{
+			Url:      imageURL,
```

Comment:
Can you give me an example image URL?

Reply:
PR descrition, Testing Performed, scanner CLI call.